integrationtests/src/test/java/io/kroxylicious/proxy/MetricsIT.java

```diff
@@ -22,50 +19,46 @@
 import io.micrometer.core.instrument.Metrics;
 import io.netty.handler.codec.http.HttpResponseStatus;
 
+import io.kroxylicious.test.tester.AdminHttpClient;
 import io.kroxylicious.testing.kafka.api.KafkaCluster;
 import io.kroxylicious.testing.kafka.junit5ext.KafkaClusterExtension;
 
-import static io.kroxylicious.proxy.Utils.startProxy;
-import static java.net.http.HttpResponse.BodyHandlers.ofString;
+import static io.kroxylicious.test.tester.KroxyliciousConfigUtils.proxy;
+import static io.kroxylicious.test.tester.KroxyliciousConfigUtils.withDefaultFilters;
+import static io.kroxylicious.test.tester.KroxyliciousTesters.kroxyliciousTester;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 @ExtendWith(KafkaClusterExtension.class)
 public class MetricsIT {
 
-    public static final String PROXY_ADDRESS = "localhost:9192";
-
     @BeforeEach
     public void beforeEach() {
         Metrics.globalRegistry.clear();
     }
 
     @Test
-    public void shouldOfferPrometheusMetricsScrapeEndpoint(KafkaCluster cluster) throws Exception {
-        String config = baseConfigBuilder(PROXY_ADDRESS, cluster.getBootstrapServers())
+    public void shouldOfferPrometheusMetricsScrapeEndpoint(KafkaCluster cluster) {
+        KroxyliciousConfigBuilder config = withDefaultFilters(proxy(cluster))
```

Comment:
Removed them, though this only works because we don't send kroxy any Kafka messages in this test. I raised an issue to make Kroxy behave more gracefully when no filters are defined #337